src/types/SplitRecipient.ts

```diff
@@ -0,0 +1,14 @@
+export interface SplitRecipient {
```

Comment:
should probably also document the top level recipient interface itself, not crucial though